Dockerfile

```
@@ -22,7 +22,8 @@ RUN useradd -m -u 1000 -U -s /bin/sh -d /pint pint && \
     mkdir /data && \
     chown -R pint:pint /data && \
     ln -s /data /pint/.local/share/filecoindot-template && \
-    rm -rf /usr/bin /usr/sbin
+    rm -rf /usr/bin /usr/sbin \
+RUN update-ca-certificates
```

```suggestion
    rm -rf /usr/bin /usr/sbin && \
    update-ca-certificates
```

can we move this `update-ca-certificates` to the top of the last `RUN` command? each `RUN` command creates a new layer in docker